### ISSUE TITLE
Add instructions to exhausted licenses to make them work for the current running project

### ISSUE THREAD

**Issue by novoselt** [sagemathinc/cocalc#7225]
One step is obvious - if the license is for 5 seats and there are already 5 projects running, it is necessary to stop one of them to free up resources to this one. (It has to be clear, however, that those 5 are OTHER projects, the current one is not included!)

Less obvious - if this project was started when the license was exhausted, but then other projects were stopped to free up resources, it is necessary to restart the current project to apply the license upgrades. To many users it may not be at all obvious!

**Comment by mforbes** [comment]
It might be a good idea to put a "Restart Project" button with a help button on the `upgrades` page, perhaps next to the "Refresh" button.  Right now, I think you need to leave the upgrades page and go to setting to restart the project.

**Comment by novoselt** [comment]
This also assumes that the user has noticed that something is not working with license upgrades and actively tries to fix the situation. It may be helpful to be even more proactive: if the project is started when one of its licenses is exhausted - display a banner explaining the situation. If an applied license becomes available to be used by this project - display a banner with a button suggesting the restart.

**Comment by mforbes** [comment]
Agreed (although sometimes banners can get a little annoying... like the ones that tell me a project has to be started in order to view a file when I am simply trying to close that tab.)

**Comment by novoselt** [comment]
Users are confused by the current behaviour and I think I would be confused too as a casual user. The problem probably got worse with some projects autostarted - if I worked on one project yesterday and today I want to work on a different one, then I open it, but the one from yesterday was already started and has used up the license. With limiting functionality of free projects, it is also important to use purchased licenses smoothly.

**Comment by novoselt** [comment]
There are alternative steps planned.
